target/linux/ramips/dts/mt7621_maginon_mc-1200ac.dts

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "maginon,mc-1200ac", "mediatek,mt7621-soc";
+	model = "Maginon MC-1200AC";
+    
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+	
+	aliases {
+		led-running = &top_blue_led;
+		led-boot = &top_red_led;
+		led-failsafe = &top_red_led;
+		led-upgrade = &top_red_led;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		top_blue_led: blue_led {
```

Comment:
The led names should start with "led".
Change blue_led to led_blue and similarly red_led to led_red.